src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2232,15 +2232,19 @@ static void _svgLoaderParserXmlOpen(SvgLoaderData* loader, const char* content,
         }
         loader->latestGradient = gradient;
     } else if (!strcmp(tagName, "stop")) {
+        if (!loader->latestGradient) {
+#ifdef THORVG_LOG_ENABLED
+            printf("SVG: Gradient stop but no gradient in the scope\n");
```

Comment:
A little ambiguous to me. Could you please write it for more user-friendly?
The logs are tip information for users (designers & developers)

ex) Stop element is used outside of Gradient element.

Reply:
Ok, done